### PR TITLE
feat: add span.Warnings to cassandra schema

### LINKED ISSUE #704: Cassandra's dbmodel's Span.SpanHash depends on data that isn't in dbmodel
The computation of `Span.SpanHash` defined in `/cassandra/spanstore/dbmodel/model.go` depends on a field (`Warnings`) which isn't in dbmodel's Span.

https://github.com/jaegertracing/jaeger/blob/a631623e52a376b98cf102c423ecd9b6fc81fdf4/plugin/storage/cassandra/spanstore/dbmodel/model.go#L41

We compute the SpanHash by using the common model as seen here:

https://github.com/jaegertracing/jaeger/blob/412baf60db217d6f41d7fdca04358069362b29ca/plugin/storage/cassandra/spanstore/dbmodel/converter.go#L40-L45

### LINKED ISSUE #705: Span.Warnings not stored in Cassandra
The `model Span.Warnings` field isn't stored in Cassandra. 

https://github.com/jaegertracing/jaeger/blob/fcbd21078ab7ed03682964c187452778242d3423/model/span.go#L59

Cassandra schema for reference:
https://github.com/jaegertracing/jaeger/blob/ec6378a7a5094ff3019c506163108da29f3f2f99/plugin/storage/cassandra/schema/v001.cql.tmpl#L54-L68

The warnings field contains details on any conversion failures from thrift to the internal model object. I feel that this is important information that should be persisted along with the rest of the Span.

### PR DESCRIPTION
**Pull request by rohankmr414**
## Which problem is this PR solving?
- Resolves #704
- Resolves #705

## Short description of the changes
- added `Span.Warnings` to Cassandra `dbmodel.Span`

### CODE REVIEW

**Comment by rohankmr414** [comment]
@albertteoh do I also need to include some sort of migration for cassandra schema

File: plugin/storage/cassandra/spanstore/writer.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ const (
 	insertSpan = `
 		INSERT
 		INTO traces(trace_id, span_id, span_hash, parent_id, operation_name, flags,
-				    start_time, duration, tags, logs, refs, process)
-		VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+				    start_time, duration, tags, logs, refs, process, warnings)
+		VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
```
[the remark this answers is not in the document]
Comment (yurishkuro) [reply plugin/storage/cassandra/spanstore/writer.go:40]: For example, we could create tags like this:
  * `$$span.warning.1 = {warning1}`
  * `$$span.warning.2 = {warning2}`

[the remark this answers is not in the document]
Comment (utsavoza) [reply plugin/storage/cassandra/spanstore/writer.go:40]: Should we be concerned about the length of the warning (string) we receive in the span?

[the remark this answers is not in the document]
Comment (yurishkuro) [reply plugin/storage/cassandra/spanstore/writer.go:40]: Alternative implementation in #4313

**Comment by yurishkuro** [comment]
Replaced by #4313

**Pull request closed without merging**